crates/capi/src/lib.rs

```diff
@@ -1,19 +1,17 @@
-//! This file defines the extern "C" API, which is compatible with the
-//! [Wasm C API](https://github.com/WebAssembly/wasm-c-api).
+//! Wasmtime's embedding C API
```

Comment:
Is there a reason for removing the URL for wasm-c-api here? I guess we do have the URL in the top-level README.md too, but since the purpose of this file is to implement that API, that would seem to be useful context here.

Reply:
Whoops, just a mistake!